tests/lib/rules/no-inner-literal.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const rule = require('../../../lib/rules/no-inner-literal');
+const {RuleTester} = require('eslint');
+
+let ruleTester = new RuleTester();
+ruleTester.run('no-inner-literal', rule, {
+  valid: [{
+    code: 'expect(a).to.be.ok;'
+  }, {
+    code: 'expect(a && b).to.be.ok;'
+  }, {
+    code: 'expect(a || b).to.be.ok;'
+  }, {
+    code: 'expect(a).to.equal(5);'
+  }],
+
+  invalid: [{
+    code: 'expect(undefined).to.be.ok;',
+    errors: [{
+      message: '`undefined` used in expect()'
+    }]
+  }, {
+    code: 'expect(null).to.be.ok;',
+    errors: [{
+      message: '`null` used in expect()'
+    }]
+  }, {
+    code: 'expect(true).to.be.ok;',
+    errors: [{
+      message: '`true` used in expect()'
+    }]
+  }, {
+    code: 'expect(52).to.be.ok;',
+    errors: [{
+      message: '`52` used in expect()'
+    }]
+  }, {
+    code: 'expect("string").to.be.ok;',
+    errors: [{
+      message: '`"string"` used in expect()'
+    }]
```

Comment:
I just remembered that backticks in JS are also a thing. can you add test cases for template literals too? probably one with and one without a prefix

Reply:
Yeah, thought about that too, but not sure we want this though because template literals allow JS expressions within them, so they are not quite "literals" in the same fixed sense...

Can be, e.g.,

```js
expect(`Hello ${place}`).to.equal('Hello world')
```

Reply:
yeah, I guess that's true. Can you still add it as a passing test case though? 🙏

Reply:
Sure, I added a regular template literal and a tagged template literal.